include/verilated.mk.in

```diff
@@ -10,7 +10,7 @@
 ######################################################################
 
 AR = @AR@
```

Comment:
Please do AR and all in this block tbrough PYTHON3, (e.g. you will need LINK) thanks.

Reply:
Done!